.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+# CircleCI 2.0 configuration file for running Python on macOS.
```

Comment:
Can we also disable email notifications of failed builds?

Reply:
I have disabled them from the web UI. I don't know how todo it from config.